templates/components/common/navigation-list.html

```diff
@@ -65,6 +64,49 @@
                     {{/if}}
                 </li>
             {{/each}}
+            <ul>
+                <li>
+                    <a 
+                        class="js-nav-pages-action"
+                        href="{{url}}"
+                        aria-label="{{lang 'category.view_all.name' category=name}}"
+                    >
+                        {{lang 'common.login'}}
+                    </a>
+                </li>
+                <li>
+                    <a 
+                        class="js-nav-pages-action sign-in-now"
```

Comment:
what is `sign-in-now` ? if it's component-related class it should start with `.c-sign-in-now`. And no BEM rules applied here as well :(